packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -10,29 +10,35 @@
  * governing permissions and limitations under the License.
  */
 
+import {
+  BreakpointProvider,
+  shouldKeepSpectrumClassNames,
+  useDOMRef,
+  useMatchedBreakpoints,
+  useStyleProps
+} from '@react-spectrum/utils';
 import clsx from 'clsx';
 import {DOMRef} from '@react-types/shared';
 import {filterDOMProps} from '@react-aria/utils';
 import {I18nProvider, useLocale} from '@react-aria/i18n';
 import {ModalProvider, useModalProvider} from '@react-aria/overlays';
 import {ProviderContext, ProviderProps} from '@react-types/provider';
 import React, {useContext, useEffect, useRef} from 'react';
-import {
-  shouldKeepSpectrumClassNames,
-  useDOMRef,
-  useStyleProps
-} from '@react-spectrum/utils';
 import styles from '@adobe/spectrum-css-temp/components/page/vars.css';
 import typographyStyles from '@adobe/spectrum-css-temp/components/typography/index.css';
 import {useColorScheme, useScale} from './mediaQueries';
 // @ts-ignore
 import {version} from '../package.json';
 
 const Context = React.createContext<ProviderContext | null>(null);
+Context.displayName = 'ProviderContext';
+
+const DEFAULT_BREAKPOINTS = {S: 380, M: 768, L: 1024};
```

Comment:
Do we wanna use these as our default breakpoints? I think I remember someone mentioning that Spectrum had `S` as 600 or something like that?

Reply:
Lets handle this in followup

Reply:
https://github.com/adobe/react-spectrum/issues/1911